README.md

```diff
@@ -1679,6 +1679,13 @@ Nice to meet you!
 Have a good day!
 ![cat](https://i.imgur.com/g3D5jNz.jpeg)
 
+---
+# Hi all!
+My name is [Dmitry](https://github.com/akulahere) and I'm glad to see you.
+Here it is the photo of my cat:
+![blizzard](assets/images/blizzard.png)
+
+
 ---
 #Hi there
```

Comment:
One more thing, please fix typo

`#Hi there` Yours :red_circle:
#Hi there

`# Hi there` will be correct :green_circle: 
# Hi there

Reply:
fixed, thank you for the hint